src/bin/conserve/show.rs

```diff
@@ -0,0 +1,687 @@
+// Conserve backup system.
```

Comment:
I think this file is now clumping together a couple of things that I would rather move towards being more separate:

1. Binding the Monitor to Nutmeg to draw text progress
2. Rendering various objects to text or json: the "bulk output" type of output. 
3. (Also listing versions, which is conceptually different from the previous point, though admittedly tangled together in the existing code.)

Emitting json through `info!` feels wrong since it will be mangled unless prefixes are turned off.

Maybe the "render to string" things should move into `fmt::Display` or something that takes a `Write`. At least for now let's keep them in a separate file?